toplevel/opttopmain.ml

```diff
@@ -20,6 +20,26 @@ let usage =
 
 let preload_objects = ref []
 
+(* Position plus length of last expand argument *)
+let expand_offset: (int * int) option ref = ref None
```

Comment:
I updated the code. It is actually simpler to just store the last position of the current expanded array.